src/Runner.Worker/action_yaml.json

```diff
@@ -201,6 +203,21 @@
             ],
             "string": {}
         },
+        "boolean-steps-context": {
+            "context": [
+                "github",
+                "needs",
+                "strategy",
+                "matrix",
+                "secrets",
```

Comment:
we need to remove secrets from this, copy the items available in `string-steps-context`

Reply:
`inputs` should also be allowed

Reply:
Thanks!

Reply:
Thank you!